public/index.html

```diff
@@ -51,7 +51,7 @@ <h1 data-translation-id="page_title" class='h1'>Twin Cities Aid Map</h1>
   </div>
   <div id="help-info" class='help-info padded spaced-lines'>
     <button id='help-info-close-button' class='help-info-close-button' aria-label="Close help info section">X <span data-translation-id="close">Close</span></button>
-    <p class='p txt-small'><span data-translation-id="project_description">Twin Cities Aid Map is run by volunteers</span>.</p><br />
+    <p class='p txt-small'><span data-translation-id="project_description">Twin Cities Mutual Aid Map is ran by volunteers as part of the <a href="http://tcmap.org">Twin Citis Mutual Aid Project</a></span>.</p><br />
```

Comment:
`is run` was correct usage here :)

Reply:
Hmm, I thought I pushed a fix for the typo!